packages/app-project/src/screens/ProjectAboutPage/components/AboutNavLink/AboutNavLink.stories.js

```diff
@@ -16,11 +16,6 @@ const mockedRouter = {
   }
 }
 
-const currentLink = {
```

Comment:
This type of testing "active link" styling doesn't work well in Storybook because we don't have a Next.js Router mock setup for all stories. I'll just remove for now.